daemon/cdi.go

```diff
@@ -117,12 +116,13 @@ func (c *cdiHandler) injectCDIDevices(s *specs.Spec, dev *deviceInstance) error
 }
 
 // getErrors returns a single error representation of errors that may have occurred while refreshing the CDI registry.
-func (c *cdiHandler) getErrors() error {
-	var err *multierror.Error
+func (c *cdiHandler) getErrors() (err error) {
```

Comment:
Small nit; I'm trying to avoid using `err` as name for named error returns. In this case it's a small function, but we've had various cases where it led to confusing code where the return error was handled in a `defer`.


Could you make it a `var err error` inside the function body?